src/_pytest/main.py

```diff
@@ -406,7 +409,14 @@ def __init__(self, config: Config) -> None:
         self._initialpaths = frozenset()  # type: FrozenSet[py.path.local]
 
         # Keep track of any collected nodes in here, so we don't duplicate fixtures
-        self._node_cache = {}  # type: Dict[str, List[Node]]
+        self._node_cache1 = {}  # type: Dict[py.path.local, Sequence[nodes.Collector]]
```

Comment:
can we use more descriptive names?

Reply:
TBF @bluetech commented as much:

> Suggestions welcome for better names than 1, 2, 3 :)

I have no good suggestions though. 😬

Reply:
I did this change mechanically, didn't really try to understand the purpose, so I don't really know which better names to use.

Reply:
sounds like a solid no then, thanks for elaborating